src/GSoft.Extensions.MediatR.ApplicationInsights/RequestApplicationInsightsBehavior.cs

```diff
@@ -1,11 +1,12 @@
-﻿using MediatR;
+﻿using System.Diagnostics;
+using MediatR;
 using Microsoft.ApplicationInsights;
 using Microsoft.ApplicationInsights.DataContracts;
 
 namespace GSoft.Extensions.MediatR;
 
 internal sealed class RequestApplicationInsightsBehavior<TRequest, TResponse> : IPipelineBehavior<TRequest, TResponse>
-    where TRequest : IRequest<TResponse>
+    where TRequest : notnull
```

Comment:
This change was required due to a breaking change in MediatR 12.
See: https://github.com/jbogard/MediatR/wiki/Migration-Guide-11.x-to-12.0